.github/issue_template.md

```diff
@@ -0,0 +1,17 @@
+**Upstream bug:** <!-- Link to the bug tracker where you have reported this issue -->
+**Date:** <!-- YYYY-MM-DD -->
+**Reported by:** <!-- Name of person that reported this bug -->
+
```

Comment:
I've also been using 

```
**Related to:** Link to corresponding issue or PR in alphagov repo, if applicable
```

if there's a PR / issue in another repo that's related – do we think that's useful?

Reply:
I figured it if was people would add it in their notes below. Wanted to keep this super minimal.

Reply:
Added it in